packages/block-library/src/latest-posts/edit.js

```diff
@@ -52,6 +52,7 @@ const CATEGORIES_LIST_QUERY = {
 };
 const USERS_LIST_QUERY = {
 	per_page: -1,
+	who: 'has_published_posts',
```

Comment:
I don't have enough exposure to the REST API to tell whether making the parameters work differently than the `WP_User_Query` is something we allow to do, so I'll leave others to comment on this aspect.

For my own education, I wanted to gather your thoughts on an alternative: what if we added support for `has_published_post`? My understanding is that in the case of the latest-posts block it could be enough if we passed `true` to the endpoint:

```js
const USERS_LIST_QUERY = {
  per_page: -1,
  has_published_post: true,
};
```

Reply:
That would also work... I was simply trying to make the changes minimally invasive, and the `who` arg seemed like a reasonable way to accomplish what we need. I'll see if I can make that work :+1:

Reply:
I think your suggestion makes more sense than what I was initially doing... Pushed a fix and now it has a `has_published_posts (boolean)` arg :+1: